kubeyard/entrypoints/kubeyard.py

```diff
@@ -128,6 +128,10 @@ def fix_code_style(**kwargs):
     "--image-context",
     help="Image context containing Dockerfile. Defaults to <project_dir>/docker",
 )
+@click.option(
+    "--docker-args",
+    help="Additional arguments passed to docker command. Use like that: --docker-args=\"-a x --arg2=y --arg3=z\"",
```

Comment:
```suggestion
    help='Additional arguments passed to docker command. Use like that: --docker-args="-a x --arg2=y --arg3=z"',
```

Reply:
Yeah, I considered that way too. Now it is implemented as you suggested.